### PR TITLE
Fixing api changes and use release tooles from api repo

### PR DESCRIPTION
**Pull request by brahmaroutu**
Build changes to build a container, change makefile to pull release tools from api repo, fix API changes that are recently affected BR BAR objects.

ping @wlan0

### CODE REVIEW

File: pkg/bucketaccessrequest/bucketaccessrequest.go [hunk 1]
```diff
@@ -126,14 +126,17 @@ func (b *bucketAccessRequestListener) provisionBucketAccess(ctx context.Context,
 
 	bucketaccess = &v1alpha1.BucketAccess{}
 	bucketaccess.Name = util.GetUUID()
+
+        bucketaccess.Spec.BucketInstanceName  = bucketRequest.Spec.BucketInstanceName
```
Comment (wlan0) [inline pkg/bucketaccessrequest/bucketaccessrequest.go]: gofmt

File: pkg/util/util.go [hunk 2]
```diff
@@ -35,3 +36,7 @@ func CopyStrings(s []string) []string {
 func GetUUID() string {
 	return string(uuid.NewUUID())
 }
+
+func ReadObject(o *v1alpha1.ObjectReference) string {
```
Comment (wlan0) [inline pkg/util/util.go:41]: Since, we're just using a stub for now, maybe add a TBD comment above this function?

File: Makefile [hunk 3]
```diff
@@ -12,7 +12,18 @@
...
+.PHONY: reltools
+reltools: download
+download:
+	$(eval CURDIR := $(shell pwd))
+	$(eval TMP := $(shell mktemp -d))
+	$(shell cd ${TMP} && git clone git@github.com:brahmaroutu/api.git)
+	$(shell cp -r ${TMP}/api/release-tools ${CURDIR}/)
+	$(shell rm -rf ${TMP})            
+
+CMDS=cosi-controller-manager ccm
```
Comment (wlan0) [inline Makefile]: would the developer have to run `make ccm` (or) `make cosi-controller-manager`

File: pkg/bucketaccessrequest/bucketaccessrequest.go (hunk 1 above)
Comment (brahmaroutu) [reply pkg/bucketaccessrequest/bucketaccessrequest.go]: done

File: pkg/util/util.go (hunk 2 above)
Comment (brahmaroutu) [reply pkg/util/util.go:41]: fixed

**Review by brahmaroutu — COMMENTED** [review]
Yes it should be just cosi-controller-manager

**Comment by wlan0** [comment]
LGTM

**Pull request merged**